src/components/file-input/examples/file-input.tsx

```diff
@@ -0,0 +1,80 @@
+import { FileInfo } from '@limetech/lime-elements';
+import { Component, h, State } from '@stencil/core';
+
+/**
+ * Basic example
+ */
+@Component({
+    tag: 'limel-example-file-input',
+    shadow: true,
+})
+export class FileInputExample {
+    @State()
+    private disabled = false;
+
+    @State()
+    private readonly = false;
+
+    @State()
+    private multiple = false;
+
+    @State()
+    private files: FileInfo[] = [];
+
+    public render() {
+        return [
+            <limel-file-input
+                onFilesSelected={this.handleFilesSelected}
+                disabled={this.disabled || this.readonly}
+                multiple={this.multiple}
+            >
+                <limel-button label="Select a file" />
+                {this.files.map((file) => (
+                    <limel-chip
```

Comment:
Similar point here about chips wrapped by the input component

Reply:
See https://github.com/Lundalogik/lime-elements/pull/2819/commits/d3850a937286572606536aa31354839dec8b30dd